main.cpp

```diff
@@ -1,10 +1,38 @@
 #include <SDL2/SDL.h>
 #include <SDL2/SDL_image.h>
 #include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
 #include "rf.h"
 
 int main(int argc, char *argv[])
 {
+    char optc;
+
+    rf_global_args.FREQ = DEFAULT_FREQ;
+    rf_global_args.SAMPLE_RATE = DEFAULT_SAMPLE_RATE;
+    rf_global_args.SYMBOL_RATE = DEFAULT_SYMBOL_RATE;
+
+    while ((optc = getopt(argc, argv, "f:s:b:")) > 0) {
+        switch (optc) {
+        case 'f':
+            rf_global_args.FREQ = atoi(optarg);
+            break;
+        case 's':
+            rf_global_args.SAMPLE_RATE = atoi(optarg);
+            break;
+        case 'b':
+            rf_global_args.SYMBOL_RATE = atoi(optarg);
```

Comment:
I just added an argument since it was with all of the other constants, and it wasn't difficult to add (even if I don't know of a use-case).  I wouldn't be surprised if there are some non-standard devices out there which break the standard, but I haven't actually seen one.

That being said, the remote for the device I was testing was a little bit off from the expected value (more like 1900), although it still worked fine with the default (I suspect there is a lot of tolerance here).

I can remove it, or I can add documentation to say that the user should not need to change this value.